tests/test_extensions/test_ext_autodoc_autoclass.py

```diff
@@ -515,3 +515,55 @@ def test_autoattribute_TypeVar_module_level(app):
         "   alias of TypeVar('T1')",
         '',
     ]
+
+
+@pytest.mark.sphinx('html', testroot='ext-autodoc')
+def test_inherited_instance_variable_with_annotations(app):
```

Comment:
No need to comment what the test does I think. Most of the tests don't have a docstring and it should be pretty clear what we are actually doing because of the name imo.

Reply:
removed